src/Microsoft.CodeQuality.Analyzers/CSharp/Maintainability/CSharpRemoveUnusedLocals.Fixer.cs

```diff
@@ -14,6 +15,17 @@ namespace Microsoft.CodeQuality.CSharp.Analyzers.Maintainability
     [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
     public sealed class CSharpRemoveUnusedLocalsFixer : RemoveUnusedLocalsFixer
     {
-        public override ImmutableArray<string> FixableDiagnosticIds => ImmutableArray.Create("CS0168");
+        public override ImmutableArray<string> FixableDiagnosticIds => ImmutableArray.Create("CS0168", "CS0219", "CS8321");
```

Comment:
Is it possible to use constants instead of magic strings here?

Reply:
Thank you! It is nice to have them declared in the corresponding analyzer. Then, sharing constants will be helpful. However, the analyzer is in the compiler code and does not provide those constants externally. Therefore, constants can be declared for local usage only. It is not so helpful to extract into constants in this case. And the current approach seem to be consistent with other usages.

---
In reply to: [149800470](https://github.com/dotnet/roslyn-analyzers/pull/1336#discussion_r149800470) [](ancestors = 149800470)